yt/visualization/tests/test_geo_projections.py

```diff
@@ -1,79 +1,58 @@
+"""
+Tests for making unstructured mesh slices
+
+"""
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# -----------------------------------------------------------------------------
```

Comment:
very quick : this smells like something went wrong during rebase or merging.

Reply:
@neutrinoceros How do you mean? Is this related to the PR removing these kinds of headers?

Reply:
yes, they were all removed, so I'm worried to see them reappearing here.

Reply:
This is one of the files in which you currently have conflicts, so I wouldn't be surprised if that was the reason.

Reply:
Ok, cool, thanks! I'll do another merge with master locally and then push, see where that gets us!